specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
The example file should be linked to the related API. then can validate if the example and swagger change are aligned.

Reply:
Sure.. I will see to it that it is done. Thanks